README.md

```diff
@@ -592,6 +592,15 @@ poetry2nix.mkPoetryApplication {
 }
 ```
 
+Please remember that `withDefaults` function, applies user overrides first, and than applies `defaults`. If you want to change package
+that already is overriden by `defaults` you may need to do 
```

Comment:
Can you rewrite it like this?
The text and the code are separated by an empty line.

>
> ```
>you can use the a snippet like the following which then puts your changes on top of the defaults:
> 
> ```nix
> ```